messaging/testapp/Assets/Firebase/Sample/Messaging/UIHandlerAutomated.cs

```diff
@@ -13,7 +13,7 @@ public class UIHandlerAutomated : UIHandler {
 
     private const string TestTopic = "TestTopic";
     private const string ServerKey = "REPLACE_WITH_YOUR_SERVER_KEY";
-    private const string FirebaseBackendUrl = "REPLACE_WITH_YOUR_BACKEND_URL";
+    private const string FirebaseBackendUrl = "https://fcm.googleapis.com/fcm/send";
```

Comment:
Is this change intentional?

Reply:
Yeah, there is no reason to obfuscate this url, it is publicly known.